src/index.ts

```diff
@@ -20,22 +20,68 @@ import mapperReader from './mapperReader.js';
 import { MapperServer } from './mapperServer.js';
 import { BidiServerRunner } from './bidiServerRunner.js';
 import { IServer } from './iServer.js';
+import argparse from 'argparse';
+
+function parseArguments() {
+  var parser = new argparse.ArgumentParser({
+    add_help: true,
+    exit_on_error: false,
+  });
+
+  parser.add_argument('-p', '--port', {
```

Comment:
Can `type: 'int'` be added here to take care of the int parsing too?

Reply:
done